### ISSUE TITLE
Tab bar should be hidden on Checkout flow

### ISSUE THREAD

**Issue by dstrukt** [btcpayserver/app#51]
## Update

All views, failure or success (whether it's a wallet/node failure or a successful payment), should allow the user to go "Back", and in most cases, likely the default Keypad view.

The ability to accidentally fat finger one of the tabs and mess up the Checkout, thus having to generate more than 1 invoice for the same sale is not ideal.

We plan to hide when we have a more "native" checkout experience instead of the current iframe, but at this moment, technically, it's too annoying to hide, so we will keep it.

## Original

Is there any reason to keep the Tab bar visible when going through the Checkout flow? To be clear, after the `Charge` CTA is clicked.

We have all of the relevant links to escape the view, and clicking away to one of the other tabs kills the flow/process anyway, so I think we should go ahead and hide it, or push it below on the `z-index` if we can't technically hide.

<img width="461" alt="Screen Shot 2024-06-26 at 2 42 51 PM" src="https://github.com/btcpayserver/app/assets/6250771/bd9e7e0d-6b2e-48c3-a404-432fed1bf163">

**Comment by pavlenex** [comment]
I mean merchant can generate an invoice, user's wallet may not work or may not be able to pay, how is merchant supposed to come back and generate an invoice for a next customer? Do you have any good examples for traditional payment checkouts which hide these in their flows?

**Comment by dennisreimann** [comment]
I'd also vote for keeping it, because otherwise we'll have to do hacks for going back from several of the server pages.

**Comment by pavlenex** [comment]
Okay I've read the design notes and saw decision was to not implement this one but keep the issue opened. However, after talking to @dennisreimann and seeing that we've removed the invoices list from the POS (which means merchant needs tab bar to navigate to `Invoices` page in case they need to troubleshoot, I've decided to close this one. If in the future we get feedback to revive this feature, we can re-open.
